test/plugin/scenarios/rocketmq-scenario/config/expectedData.yaml

```diff
@@ -59,7 +59,7 @@ segmentItems:
             componentId: 39
             isError: false
             spanType: Entry
-            peer: ''
+            peer: not blank
```

Comment:
So, peer would be `namesrvAddr`, which is the full list used for MQ client. Am I right?

Reply:
Yes, you are right.

Reply:
I think we don't need to  set peer at consumer side .  Right  ? 
`virtual mq `  organize   ` peer `( at produce side) , `networkAddressUsedAtPeer ` ( at  consume side ) as service name  .  If we set correct peer at produce side ,   it  would be propagated to consumer with carrier .

Refer  to  https://github.com/apache/skywalking/pull/9855#issuecomment-1297187181

Reply:
Yes, we don't have to, but still, be good to have it on the peer.

@zhyyu If you want to reduce resource cost, feel free to `skip` peer at the consumer side.

Reply:
emmm  , We  got a  bug  , Let's keep peer existing  in this  plugin . 
And fix other mq plugin  , OAP  in another PR

Reply:
Not  bad  . It needs more care to confirm if plugins set correct peer  when reviewing